### ISSUE TITLE
get text by language fails

### ISSUE THREAD

**Issue by raduangelescu** [c-w/Gutenberg#70]
Populated my cache with the default berkley db but when I do 
```python
eng = get_etexts('language','en') 
or get_etexts('language','eng')
or get_etexts('language','En')
```
The query returns the empty frozenset. 
 
Is this a known bug?

**Comment by c-w** [comment]
The language information is stored as [dcterms:language](https://terms.tdwg.org/wiki/dcterms:language) in the Project Gutenberg metadata which recommends RFC 4646 to specify the languages, so tags like *en*, *fr*, *fi*, *de*, etc. (which you used) should be the correct way to query the data-set. Doing a quick grep through the RDF files confirms that you executed the right query, so this must be a bug.

**Comment by c-w** [comment]
I can reproduce this, adding `bug` label.

**Comment by c-w** [comment]
I found the root cause of this. The language metadata extractor was implemented incorrectly back in #50. The language field in the RDF is typed as RFC 4646 so we need to reflect this in the extractor.

Example RDF:

```xml
<dcterms:language>
  <rdf:Description rdf:nodeID="Ndc98bbe6c7bb4281b75c633906a61a97">
    <rdf:value rdf:datatype="http://purl.org/dc/terms/RFC4646">nl</rdf:value>
  </rdf:Description>
</dcterms:language>
```

The important part here is the `datatype="http://purl.org/dc/terms/RFC4646"`. We'll need to specify this when converting the queried value into RDF. This can be done via the datatype argument to the [Literal constructor](http://rdflib.readthedocs.io/en/stable/rdf_terms.html#literals).

**Comment by c-w** [comment]
@raduangelescu: Thanks so much for reporting this. The bug should now be fixed. Could you please install the latest gutenberg version from the repository and let me know if the problem is fixed for you? If it's not, please feel free to re-open the issue.
